sdk/communication/Azure.Communication.CallingServer/tests/CallRecording/ContentDownloadLiveTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Communication.CallingServer.Tests.Infrastructure;
+using NUnit.Framework;
+
+namespace Azure.Communication.CallingServer
+{
+    public class ContentDownloadLiveTests : CallAutomationClientLiveTestsBase
+    {
+        public ContentDownloadLiveTests(bool isAsync) : base(isAsync)
+        { }
+
+        [Test]
+        public async Task DownloadMetadata()
+        {
+            if (SkipCallingServerInteractionLiveTests)
+            {
+                Assert.Ignore("Skip callingserver interaction live tests flag is on");
+            }
+
+            CallAutomationClient client = CreateInstrumentedCallingServerClientWithConnectionString();
+            string documentId = "0-wus-d4-d4c223871b58d3664401b66d35fca784";
```

Comment:
is this sensitive data?

Reply:
it is not, but I could replace it to prevent any concern.

Reply:
oh dont worry about it if it's not sensitive data.